Core/bin/README.md

```diff
@@ -0,0 +1,73 @@
+Scripts
+==========
+
+Scripts modules first of all loads the environment variables by the `getenv.sh` script. Some of the most important variables that are needed are:
+
+- MA_ENV_PROPERTIES
+- MA_ENV_PROPERTIES
```

Comment:
Duplicate

Reply:
Remove all references to MA_ENV_PROPERTIES anyway, this should no longer be used and is replaced by `mango_config`

Reply:
I removed the duplicate one and remove the references for MA_ENV_PROPERTIES